icu4c/source/i18n/unicode/calendar.h

```diff
@@ -2169,7 +2169,7 @@ class U_I18N_API Calendar : public UObject {
     /**
      * The next available value for fStamp[]
      */
-    int32_t fNextStamp;// = MINIMUM_USER_STAMP;
+    int8_t fNextStamp;// = MINIMUM_USER_STAMP;
```

Comment:
The constant is spelled kMinimumUserStamp.
Why are we not initializing it here? Why in the constructors? Hold-over from old C++ versions?
Should we simplify the constructors by pulling the trivial field initializers into the field declarations?

Reply:
I change this.
Also, I change some private UBool to bool and use bit fields to save more space. But then I found bit fields does support initializing via field description until C++20 so for those bit fields I still keep the old C++ style.